infrastructure/start_all_local/clean_startup.sh

```diff
@@ -4,4 +4,4 @@ docker image prune -f
 docker image rm start_all_local_marketplace-service:latest start_all_local_admin-ui:latest start_all_local_auth-service:latest start_all_local_client-ui:latest
 docker volume rm am_db --force
 docker volume rm keycloak_db --force
-docker-compose up
\ No newline at end of file
+docker-compose up -d 
```

Comment:
May be leave "docker-compose up" in non-detached mode?

Reply:
the paramater -d is optional. We dont neccessary need it. A matter of preference.